Python/ceval_macros.h

```diff
@@ -282,6 +290,9 @@ GETITEM(PyObject *v, Py_ssize_t i) {
 #define CONSTS() _PyFrame_GetCode(frame)->co_consts
 #define NAMES() _PyFrame_GetCode(frame)->co_names
 
+/* 1 + the cache size of SEND/FOR_ITER_GEN (they must be same) */
+#define NEXT_INSTR_OFFSET_FOR_YIELD 2
```

Comment:
Isn't there a constant for said cache size? Otherwise if it changes, this macro breaks.

Reply:
This is a constant which is equal to (1+) two cache sizes - of SEND and FOR_ITER_GEN (which are equal). There are assertions in the code that will fail if either cache size changes.